sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -57,6 +57,20 @@ public static void Main()
                 .Select("Room", "Temperature")
                 .From(AdtCollection.DigitalTwins)
                 .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE TEMPERATURE < 5
+            AdtQueryBuilder query6 = new AdtQueryBuilder()
```

Comment:
Consider naming your query builders with a meaningful name. So, `queryWithComparisonWhereClause` instead of `query6`.

Reply:
```suggestion
            var queryWithComparisonWhereClause = new AdtQueryBuilder()
```